### ISSUE TITLE
Rename `use_either_n` feature and add documentation

### ISSUE THREAD

**Issue by plul** [cramertj/domafic-rs#12]
Say I have two pages, and I want some condition to determine which one is rendered.
Essentially, what I'm looking for is a way to write if/else inside the render function. (In Angular terminology, I'm looking for an `*ngIf` equivalent). The docs and examples do not cover this.

The following example illustrates my best solution at present, using a range and a filter. However, it feels hacky and I am wondering whether something cleaner is intended.

```
    let render_page1 = |state: &State| h1(("This is page 1"));

    let render_page2 = |state: &State| h2(("This is page 2"));

    let render = |state: &State| {
        div((
            button((on("click", |_| Msg::ToggleMyBoolean), "Toggle myBoolean")),
            (0..1)
                .filter(|_| state.myBoolean)
                .map(|_| render_page1(state))
                .collect::<Vec<_>>(),
            (0..1)
                .filter(|_| !state.myBoolean)
                .map(|_| render_page2(state))
                .collect::<Vec<_>>(),
        ))
    };
```

By the way, great work on this crate! It is quite fun to play with. Cheers!

**Comment by cramertj** [comment]
You can use the `EitherN` enums from the either-n crate for this. Here's a [test](https://github.com/cramertj/domafic-rs/blob/master/src/lib.rs#L314) showing how.

Glad you're having fun playing around-- let me know if you have any other questions.

**Comment by cramertj** [comment]
Note to self: rename the `use_either_n` feature to just `either_n` so that it will automatically be included when users import the `either_n` crate.

**Comment by cramertj** [comment]
I'm actually going to reopen this until I've renamed the feature and added examples.

**Comment by plul** [comment]
This will do. Thanks!

(It does not work if I write `#[cfg(feature = "use_either_n")]`, but if I omit it, everything works).

**Comment by cramertj** [comment]
@plul Yeah, `use_either_n` is a default feature of Domafic-- it's not something you'd `cfg` in your code.

**Comment by plul** [comment]
Using your `either_n` solution works great if I want to render one page or the other, but does domafic support some kind of None tag? 

Say I want to do this:

```
if myCondition {
    either_n::Either2::One(div("Hello"))
}
else {
    either_n::Either2::Two(None)
}
```

I could of cause just do `either_n::Either2::Two("")` - it will just leave some empty `<span>`s here and there.
